dask_cuda/proxify_device_objects.py

```diff
@@ -264,3 +264,12 @@ def proxify_device_object_cudf_dataframe(
         return proxify(
             obj, proxied_id_to_proxy, found_proxies, subclass=FrameProxyObject
         )
+
+    try:
+        from dask.array.dispatch import percentile_lookup
+
```

Comment:
```suggestion
```

Reply:
Looks like `isort` isn't happy here without a space, hence had to revert this change